### ISSUE TITLE
both new and old weights don't load

### ISSUE THREAD

**Issue by jcpeterson** [titu1994/Wide-Residual-Networks#17]
Using keras 2.2.0. I tried both the new and old weights in the main 28-8 script, and neither load correctly. 

`ValueError: Layer #0 (named "conv2d_1" in the current model) was found to correspond to layer convolution2d_29 in the save file. However the new layer conv2d_1 expects 1 weights, but the saved weights have 2 elements.`

The keras contrib version of this code does load the new weights, but the accuracy on the test set is around 30%. Am I missing something?

**Comment by FrancescoCrecchi** [comment]
Indeed! Knowing the **exact** Keras version you used in your experiments would help a lot!
Any chance for this?

**Comment by titu1994** [comment]
I don't precisely remember, but it's most definitely Keras 1.x later releases.

**Comment by persistz** [comment]
Keras 2.1.3 has the same problem, both on 28-8 and 16-8 model.
